recipes/pentapy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pentapy" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a242f06ce9b2a806e544dbbbb497c3ae541e76d366114564cf877cb8b82f0bfd
+
+build:
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
```

Comment:
Normally, this shouldn't be needed as `setuptools_scm` bakes in the version into the sdist.

Reply:
ok, I'll remove the pretended version, but it shouldn't harm the building.